tasmota/language/bg-BG.h

```diff
@@ -663,6 +663,7 @@
 #define D_SENSOR_CC1101_GDO0   "CC1101 GDO0"
 #define D_SENSOR_CC1101_GDO2   "CC1101 GDO2"
 #define D_SENSOR_HRXL_RX       "HRXL Rx"
+#define D_SENSOR_ELECTRIQ_MOODL "ElectriQ MOODL Tx"
```

Comment:
Text too long. See remark in file:
```
// tasmota_template.h - keep them as short as possible to be able to fit them in GUI drop down box
```
Just drop ElectriQ and keep ``MOODL Tx`` in all language files